app/code/Magento/Store/Model/Store.php

```diff
@@ -766,7 +766,7 @@ public function isUseStoreInUrl()
      *
      * @return  int
      */
-    public function getId()
+    public function getId() :int
```

Comment:
There is backward incompatible changes due to due to `@api` mark for class.
In additional return type declaration don't fix problem and type should be converted in the body of the function `return (int)$this->_getData(self::STORE_ID);`